.tanzu/tanzu_develop.py

```diff
@@ -0,0 +1,69 @@
+load('ext://local_output', 'local_output')
+load('ext://file_sync_only', 'file_sync_only')
+# -*- mode: Python -*-
+
+# live reload
+def tanzu_develop(k8s_object, deps=["."], resource_deps=[], live_update=[]):
+
+    # 1. Create a CRD that we can create instances of that act as Tilt's image target
+    twp_crd = """
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: tiltworkloadproxys.experimental.desktop.io
```

Comment:
it's not a good practice to created CRDs under domains you don't control.

I often use the `.local` tld for experimental work

Reply:
Fair enough.  This CRD is a bit of a hack to workaround some tilt behaviors.  The real solution here is to work with tilt to make some of their behaviors customizable. 

But in the meantime I am also fine putting this in whatever domain is the "defacto standard"

Reply:
Updated from `desktop.io` to `desktop.local`